index.html

```diff
@@ -2172,8 +2174,9 @@ <h2 id="AudioContext">
           </tbody>
         </table>
         <pre class="idl">
-[Constructor,
- Constructor(optional AudioContextOptions contextOptions)]
+[Exposed=Window,
+ Constructor,
```

Comment:
Not for this PR, but should we get rid of this Constructor? Since the next constructor has a optional dictionary, aren't they equivalent?

Reply:
Yes.